salt/modules/consul.py

```diff
@@ -140,14 +154,15 @@ def list_(consul_url=None, key=None, **kwargs):
         function = 'kv/{0}'.format(key)
 
     query_params['keys'] = 'True'
-
+    query_params['separator'] = '/'
     ret = _query(consul_url=consul_url,
                  function=function,
+                 token=token,
                  query_params=query_params)
     return ret
 
 
-def get(consul_url=None, key=None, recurse=False, decode=False, raw=False):
+def get(consul_url=None, key=None, token=None, recurse=False, decode=False, raw=False):
```

Comment:
Same comment here. This new kwarg needs to be added to the end of the kwarg list.